terraform/variables.tf

```diff
@@ -0,0 +1,62 @@
+variable "aws_region" {
+  type        = string
+  description = "Default AWS region to use"
+  default     = "us-east-1"
+}
+
+variable "aws_access_id" {
+  type        = string
+  description = "AWS access ID"
+}
+
+variable "aws_secret_key" {
+  type        = string
```

Comment:
Probably need to mark some of these as sensitive

Reply:
done